applications/fluid_structure_interaction/cylinder_with_flag/CMakeLists.txt

```diff
@@ -30,3 +30,5 @@ TARGETNAME(TARGET_NAME ${CMAKE_CURRENT_SOURCE_DIR})
 PROJECT(${TARGET_NAME})
 
 EXADG_PICKUP_EXE(solver.cpp ${TARGET_NAME} solver)
+
+ADD_SUBDIRECTORY(tests)
```

Comment:
Can you please insert a line break at the end of the file.

Reply:
sure, is something like [this](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file) the reason?